test/integration/features/addons/addon-htpasswd-identity-provider.feature

```diff
@@ -0,0 +1,86 @@
+@addon-htpasswd-identity-provider @core
+Feature: Configures single identity for OpenShift login instance
+This add-on configures OpenShift to use only a single identity provider
+
+  Scenario: User can enable the htpasswd-identity-provider
+     When executing "minishift addons enable htpasswd-identity-provider" succeeds
+     Then stdout should contain "Add-on 'htpasswd-identity-provider' enabled"
+
+  Scenario: User can start Minishift with htpasswd-identity-provider add-on enabled
+    Given Minishift has state "Does Not Exist"
+      And image caching is disabled
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+      And stdout should contain
+      """
+      Successfully installed addon htpasswd identity provider
+      """
+
+  Scenario: User can login to openshift instance only with default username "developer" and password "developer"
+    Given Minishift has state "Running"
+      And user waits "60" seconds
```

Comment:
why 60 seconds!?!

Reply:
Because not sure on how much time the affected containers takes to be up and running. Might be more or less than 30 sec. So i put 60 sec to be on the safer side, otherwise make it fail.

We have similar kind of optimization done in registry route feature file, However in this case it difficult to get the service status and the responsiveness of all the affected containers. For a better resolution I mailed to aos-devel, but no response yet. Pasting the mail body 

```
In OKD 3.10 oc cluster up deploying almost 25 default containers. Is there any way or might be a REST call that confirms the service of any container from the default list is up and ready to use ? 

For example 
$ docker stop $(docker ps -l -q --filter "label=io.kubernetes.container.name=apiserver")
$ docker stop $(docker ps -l -q --filter "label=io.kubernetes.container.name=api")
$ oc login -u developer -p developer 

An immediate oc login will fail because it need some time to get the container up and running, however theses containers will be up and running in few seconds but oc login fails again and again for some more time. Probably some service in the container need to be auto fixed which takes some more time though container is up and running. 

As I asked above how can I track that time between container up time and when oc login starts working 
```

Reply:
I think idly waiting should be improved upon. It wastes a lot of time... while in the meantime we could check for liveliness and break out of the wait state early when the service actually responds... no all these 60secs add up to a test run of over 10 mins ?

Reply:
Will check